edubots_surveys.Rnw

```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
 and Differ \cite{differ.chat} facilitate breaking the ice between new students
 by introducing them to each other. Ranoliya et al. \cite{ranoliya2017chatbot}
-proposed a generic chatbot for university students that is able to answer to
-their frequently asked questions \textcolor{black}{and, for some time, the University of Granada had
-\textcolor{black}{Elvira} as a chatbot in its main web page \cite{MOREO20129} to perform that duty.} Besides being able
-to answer pre-established (frequent) questions, it did so from the website of
-the University of Granada using an inset \textcolor{black}{persona, voiced by a real person, who
-lip-synced the answers. The emphasis in the case of Elvira, however, was on the
-authenticity of the synchronization more than in making the update of questions
-answered simpler, or more interactive, or more open to the rest of the university staff, using actual messaging platforms. In
+proposed a generic chatbot for university students that is able to answer 
+frequently asked questions. The University of Granada had
+Elvira, a built-in chatbot in its main web page \cite{MOREO20129} to perform the same task. In addition to being able
+to answer pre-established frequent questions, it did so from the website of
+the University of Granada using an inset persona, voiced by a real person, who
+lip-synced the answers. However, in the case of Elvira, the emphasis was on the
+authenticity of the synchronization rather than making the updated 
+of answered questions simpler, more interactive, or more open to the rest of the university staff using actual messaging platforms. In
 most cases, the embedded search engine was able to provide more up-to-date
-and accurate answers than the ones provided by Elvira, which had to be updated
-by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide an useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, something which is the focus of this paper}.
+and accurate answers than those provided by Elvira, which had to be updated
+by hand (and not too often). It was eventually discontinued, and its technology was not adapted to other platforms, since it did not really provide a useful service. Its service was not tied to teaching anyway; it was more related to directory queries and university-wide administrative questions, so even if it is strictly an example of a chatbot in a university setting, it is not really a chatbot that can be adopted by teachers to affect the learning outcomes, which is the focus of this paper.
 
 On the other hand, teaching-oriented chatbots are more sophisticated, as they
```

Comment:
more sophisticated than what?

Reply:
I guess we meant more sophisticated than the chatbots discussed in the previous paragraph, that were focused on administrative tasks.

Reply:
Can you please add that to the text?